samples/musicfestival-frontend-react/src/pages/SearchPage.tsx

```diff
@@ -35,57 +41,90 @@ function SearchPage() {
     let autocompleteData : ArtistAutocompleteQuery | undefined = undefined
     let resultNumber : number
     let otherResultNumber : number
+    let variables: any
+    let options: string[] = ["ASC", "DESC"]
+    let itemsPerPageOptions: number[] = [10, 15]
+    let filterByOption: {value: string; key: string}[] = [
+        {value: "Artists", key: "Artist"},
+        {value: "Other Content", key: "OtherContent"}
+    ]
     
-    let modeEdit = isEditOrPreviewMode()
-    let variables = generateGQLSearchQueryVars(token, window.location.pathname, queryString, orderBy);
-    let endOffset = itemOffset + itemsPerPage
-    let endOffsetOther = otherItemOffset + otherItemsPerPage
+    let headers = {}
+    let url = singleKeyUrl
 
-    const { data : searchQueryData } = useArtistSearchQuery({ endpoint: singleKeyUrl }, variables, { staleTime: 2000, enabled: !modeEdit || !!token });
+    const { mutate } = useMutation((obj: any) => obj, {
+        onSuccess: (message: ContentSavedMessage) => {
+            if (previousSavedMessage !== message) {
+                previousSavedMessage = message;
+                updateSearchQueryCache(queryClient, artistData, variables, message)
+            }
+        }
+    })
+
+    useEffect(() => {
```

Comment:
same with [this comment](https://github.com/episerver/content-graph-js-sdk/pull/35/files#r1252835327)